tests/test_dragonnet.py

```diff
@@ -0,0 +1,18 @@
+from causalml.inference.tf import DragonNet
+from causalml.dataset.regression import *
+import shutil
+
+
+def test_save_load_dragonnet():
```

Comment:
This test requires `TensorFlow`, which is optional. To make the default build light, can we make this optional? ref: https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option